test/wdio/global-script.stencil.config.ts

```diff
@@ -0,0 +1,19 @@
+import { sass } from '@stencil/sass';
+import type { Config } from '../../internal/index.js';
+
+/**
+ * This Stencil configuration is only for the `global-script/` directory.
+ */
+export const config: Config = {
+  namespace: 'TestGlobalScript',
+  tsconfig: 'tsconfig-global-script.json',
+  outputTargets: [
+    {
+      type: 'www',
+      dir: `www-global-script`,
+    },
+  ],
+  plugins: [sass()],
```

Comment:
Do we need the `sass()` plugin here? I don't see any stylesheets here, but I could be missing some other reason for this 🤔

Reply:
ah! yeah we don't need it, I think in an earlier version of the diff I got an error without it but I must have since removed whatever was causing an error